src/components/NewThought.tsx

```diff
@@ -78,11 +79,24 @@ const NewThought = ({ path, showContexts, label, value = '', type = 'bullet' }:
         {type === 'bullet' ? <span className='bullet' /> : null}
         <div className='thought'>
           <a
-            className={classNames({
-              placeholder: type === 'bullet',
-              button: type === 'button',
-              'button-variable-width': type === 'button',
-            })}
+            className={
+              type === 'bullet'
+                ? css({
+                    fontStyle: 'italic',
+                    color: {
+                      base: 'rgba(7, 7, 7, 0.5)',
+                      _dark: 'rgba(255, 255, 255, 0.5)',
+                    },
+                  })
+                : cx(
+                    anchorButton({
+                      variableWidth: true,
+                    }),
+                  )
```

Comment:
Don't worry about it for this PR, but in the future try to write objects that only have one or two properties on a single-line like this:

```ts
cx(anchorButton({ variableWidth: true }))
```

Prettier will write them on multiple lines if needed.

Expressions with conditional styles like the `css` statement above are more readable on multiple lines though, as I would continue formatting them as you have been.